src/app/rundown-view/services/action-trigger-producer-key-binding.service.ts

```diff
@@ -17,12 +17,12 @@ import { ActionService } from '../../shared/abstractions/action.service'
 import { KeyboardTriggerData } from 'src/app/shared/models/keyboard-trigger'
 
 // TODO: We need ONE central place to define colours
-const CAMERA_COLOR: string = '#005919'
-const REMOTE_COLOR: string = '#ac29a5'
-const SPLIT_SCREEN_COLOR: string = '#00a99c'
-const REPLAY_COLOR: string = '#8d1010'
-const VIDEO_CLIP_COLOR: string = '#1769ff'
-const GRAPHICS_COLOR: string = '#ca9d00'
+const CAMERA_COLOR: string = 'var(--tv2-camera-color)'
+const REMOTE_COLOR: string = 'var(--tv2-remote-color)'
+const SPLIT_SCREEN_COLOR: string = 'var(--tv2-split-screen-upper)'
+const REPLAY_COLOR: string = 'var(--tv2-replay-color)'
+const VIDEO_CLIP_COLOR: string = 'var(--tv2-video-clip-color)'
+const GRAPHICS_COLOR: string = 'var(--tv2-graphics-color)'
```

Comment:
Since we know have the central place for the colors, then it would be nice to delete this if possible.
The TODO comment should atleast go.

Reply:
We get the color from this central place, but if I understand the logic correct we want to set background color base on constant value. I don't see the way to rework that now. I do that part in action-triggers that way:

```
<div
  (click)="selectNewAction()"
  class="c-sofie-card c-action-trigger-box"
  [ngClass]="{ 'c-action-trigger-selected': isSelected }"
  **[class]="this.action.metadata.contentType.toLocaleLowerCase() + '_color'"**
  [title]="action.name + '(' + action.type + ')'">
  {{ action.name }}
</div>
```
That way we have a class as camera_color and I got central scss with give --tv2-camera-color value to this class, but I need deep dive to the logic to see is that is possible here or not. I think that must to be out of the scope from this task

Reply:
Lets leave it for now.
Would still be nice to remove the TODO comment

Reply:
Done